youtube_transcript_api/test/test_api.py

```diff
@@ -53,11 +53,11 @@ def test_get_transcript__correct_language_is_used(self):
     def test_get_transcript__fallback_language_is_used(self):
         httpretty.register_uri(
             httpretty.GET,
-            'https://www.youtube.com/api/timedtext',
-            body=''
+            'https://www.youtube.com/watch',
+            body=load_asset('youtube_ww1_nl_en.html.static')
```

Comment:
I am still a bit confused by the ww1, but I generally prefer this name 😊

Reply:
ww1 cause its a video about some ww1 history, probably could be better named though

Reply:
aaah, okay 😄